src/mongo/models.js

```diff
@@ -6,6 +6,10 @@ const BRIDGE_CLAIM = 'claims';
 
 const EVENT_PROCESSED = 'events';
 
+const WITHDRAW_PROOF = 'withdraw_proof';
+
+const EVENT_PROOF = 'event_proof';
```

Comment:
nit: convention to use camel case for these collection names
`withdrawProof` & `eventProof`